analytics/src/main/webapp/WEB-INF/jsp/index.jsp

```diff
@@ -38,13 +38,18 @@ String forcedLang = request.getParameter("lang");
 String lang = (String) request.getAttribute("defaultLanguage");
 if (forcedLang != null) {
     if (forcedLang.equals("en") || forcedLang.equals("es") || forcedLang.equals("ru") || forcedLang.equals("fr") || forcedLang.equals("de")) {
-        lang = forcedLang;
-    }
+            lang = forcedLang;
+    } else if ( forcedLang.equals("nl") || forcedLang.equals("nl-BE") || forcedLang.equals("nl_BE")) {
+        lang = "nl";
+        }
 } else {
     if (detectedLanguage.equals("en") || detectedLanguage.equals("es") || detectedLanguage.equals("ru") || detectedLanguage.equals("fr") || detectedLanguage.equals("de")) {
         lang = detectedLanguage;
+    } else if ( detectedLanguage.equals("nl") || detectedLanguage.equals("nl-BE") || forcedLang.equals("nl_BE")) {
+        lang = "nl";
```

Comment:
Please keep the general logic here...

Reply:
Yes, I was thinking about that (though it works) and I thought about another way of testing, it looks better to my eyes, what do you think ? I think the logic is maintained, isn't it ?

```
if (forcedLang != null) {
	switch(forcedLang) {
		case "en":
		case "fr":
		case "es":
		case "ru":
		case "de":
		case "nl":
			lang = forcedLang;
			break;
		case "nl-BE":
		case "nl_BE":
			lang = "nl";
			break;
		default:
	}
} else {
	switch(detectedLanguage) {
		case "en":
		case "fr":
		case "es":
		case "ru":
		case "de":
		case "nl":
			lang = detectedLanguage;
			break;
		case "nl-BE":
		case "nl_BE":
			lang = "nl";
			break;
		default:
	}
}
```

Reply:
Is this what you're looking for ? ("nl" test moved)

I can't find a better way since "nl-BE" and "nl_BE" have to be tested further

Reply:
Not really.
I would not have expected to specialcase the `nl` lang as you're doing here...